pkgs/development/python-modules/opentracing/default.nix

```
@@ -0,0 +1,27 @@
+{ stdenv, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "opentracing";
+  version = "2.2.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "cfd231ba5c58f90bc277787e62861eb0c6e4af76e42957bec240bbdf71fb7e0e";
+  };
+
+#  preInstall = ''
+#    virtual-env env
+#    ../env/bin/activate
+#    make bootstrap
+#    make test
+#  '';
+
+  doCheck = false;
+
+  meta = with stdenv.lib; {
```

```suggestion
  meta = with lib; {
```
You can just use lib here if you put `lib` instead of `stdenv` in the function arguments at the top

---

@alexarice Done.